### DISCUSSION TITLE
Icon Request: currency Ruble

### DISCUSSION THREAD

**Discussion by denisgromov** [tailwindlabs/heroicons#381] [Ideas]

**Comment by devshok** [comment]
I created two sets for the ruble currency:

1. "currency-ruble"
2. "document-currency-ruble"

[ruble.zip](https://github.com/user-attachments/files/22734468/ruble.zip)

**p.s.** I am not a professional designer, just a software developer as many of you but if you like, you could consider my two sets to adopt and add into heroicons family.

Cheers, Ivan.

FYI @denisgromov it might not be a rapid solution due to the date of your request but it's better to suggest you new sets than nothing to do :]
